boards/nrf52dk/Makefile.include

```diff
@@ -14,7 +14,7 @@ export JLINK_DEVICE := nrf52
 
 # special options when using SoftDevice
 ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
-export JLINK_PRE_FLASH := loadfile $(BINDIR)/softdevice.hex
+export JLINK_PRE_FLASH := erase\nloadfile $(BINDIR)/softdevice.hex
```

Comment:
Humm .. what is your system ?
I have tried this on my Ubuntu 14.04.
This may be a side effect of the shell as the JLink.sh is executed by a sh -c.
My system has /bin/sh pointing to dash.
Can you check on your system ? ls -l `which sh` ?